test/e2e/fixtures.go

```diff
@@ -474,3 +475,23 @@ func createTestBusyboxPods(tb testing.TB, data *TestData, num int, ns string, no
 
 	return podNames, podIPs, cleanupFn
 }
+
+// testServiceFromNode is a helper function which uses polling mechanism to make sure that
+// Service has been realized if created and Service is no longer accessible if deleted.
+func testServiceFromNode(t *testing.T, url, nodeName string, timeout time.Duration, statusReady bool) error {
+	cmd := fmt.Sprintf("curl -s --connect-timeout 1 %s", url)
+	err := wait.Poll(1*time.Second, timeout, func() (bool, error) {
+		rc, stdout, stderr, err := RunCommandOnNode(nodeName, cmd)
+		if err != nil {
+			t.Logf(cmd)
+			t.Logf("stdout: %s - stderr: %s - err: %v", stdout, stderr, err)
```

Comment:
These logs can be merged into one with formating. e.g. 
`t.Logf("Failed to run command %q on node %q: stdout: %s - stderr: %s - err: %v", cmd, nodeName,  stdout, stderr, err)"`

Reply:
done, thanks.

Reply:
Sorry, the `node` should be `Node` in logging for consistency, and it is terminology for Kubernetes.

Reply:
done, will keep in mind from now on.Thank you.